frontend/src/app/[locale]/formPrototype/[applicationId]/[formId]/page.tsx

```diff
@@ -63,11 +62,9 @@ async function FormPage({ params }: formPageProps) {
   return (
     <GridContainer>
       <BetaAlert />
-      <h1>
-        Form Demo: Applicaction for Federal Assistance SF 424 - Individual
-      </h1>
+      <h1>Form demo for &quot;{form_name}&quot; form</h1>
```

Comment:
Has there been any thought to how we'll manage translation for something like this? I imagine maybe we just won't bother translating form pages?

Reply:
There are no designs and this will all get thrown out.